src/views/application/index.vue

```diff
@@ -224,6 +234,56 @@ const getToastOptions = (notificationInstance, toastId) => {
     },
   };
 };
+NotificationsBus.on(
+  'removePushNotificationAfterSidebarAction',
+  (notificationInstance) => {
```

Comment:
we shoul have function and pass to param for subscribing, and for unmound unsubscribe it